### ISSUE TITLE
Compare expenses made with lodging against official prices of rooms

### ISSUE THREAD

**Issue by Irio** [okfn-brasil/serenata-de-amor#26]
Filtering quota's dataset by records with value 'Lodging, except for congressperson from Distrito Federal' in the column `subquota_description` will return many expenses made with hotels. We could match the value in the receipt against publicly available (through Booking.com, for instance) range of prices.

**Comment by JVUnderground** [comment]
Holidays and events should be considered as they usually significantly the price of rooms.

**Comment by samuelgrigolato** [comment]
Does anybody have an idea how to proceed with this scraping? I mean, in addition to what is already being done by @Lrcezimbra at #100.

I had a look into _booking.com_ but couldn't find any suitable API. I also tried _decolar.com_ (they *do* have a public and _free_ API [1]), but their terms of usage doesn't seem to allow the kind of data scraping we need (I don't even know why I thought it would :smile:).

[1] http://dev.despegar.com/howto/hotels

**Comment by ebonet** [comment]
I don't believe there are historical databases for pricing. What could be done is to identify hotels on the database and start watching booking/expedia/... and scrape data, building Serenata's own dataset for that. Keep in mind that hotel pricing is somewhat complex, and database can become large.

**Comment by evilasiov** [comment]
From 2012 to now, housing pricing almost not changed.

**Comment by cuducos** [comment]
Closed accidentally by unrelated commit from Rosie/Jarbas repos.
